demosplan/DemosPlanCoreBundle/EventListener/CheckPermissionListener.php

```diff
@@ -72,21 +73,10 @@ public function onControllerRequest(ControllerEvent $event)
         // Method
         $reflectionMethod = $reflectionClass->getMethod($methodName);
         try {
-            /** @var DplanPermissions $dplanPermissions */
-            $dplanPermissions = $this->reader->getMethodAnnotation($reflectionMethod, DplanPermissions::class);
-
-            if (null === $dplanPermissions) {
-                $className = $controller::class;
-                trigger_error(
-                    "{$className}::{$methodName} does not use the @DplanPermissions annotation yet",
-                    E_USER_DEPRECATED
-                );
-
-                return;
-            }
+            $dplanPermissions = $this->getDplanPermissions($reflectionMethod);
 
             // perform initialize with permissions from annotation
-            $this->initializeService->initialize($dplanPermissions->getPermissions());
+            $this->initializeService->initialize($dplanPermissions);
```

Comment:
In initialize the function checkPermissions() is called. In checkPermissions() a log message seems not matching too.